std_msgs/Header header
float64 latitude
float64 longitude
float64 altitude
float64 utm_easting
float64 utm_northing
float64 zone
string letter_field
